### DISCUSSION TITLE
exec hive by shell but display exec failed

### DISCUSSION THREAD

**Discussion by ChanKamShing** [apache/dolphinscheduler#6360] [Q&A]
When I execute hive through shell on DS, There display failed exec on UI, but in fact, it exec successfully and has no error and I can also search in hive table . Anybody give some help?Thx.
![image](https://user-images.githubusercontent.com/48425984/134803339-b13c0ed5-4e9b-493c-9746-8b73f4fe96ff.png)
![image](https://user-images.githubusercontent.com/48425984/134803379-3553eb70-d043-4ae7-a6e9-e3ace1537acd.png)

**Comment by davidzollo** [comment]
DolphinScheduler will check two results for hive task:
```
1、the result for the task self, if the task self running result shows the exitStatusCode=0, it means this task runs successfully
2、the final task status on the yarn cluster for this hive task, DolphinScheduler will get the yarn status for this hive task through yarn http api which was configured in the `conf/common.properties` 
```
Only if these two results are successful, the hive task on the UI will show success

So if this task is judged to be failed, you should also check the runtime log of the worker server, usually the worker server will display some abnormal messages for this task
